doaj/software-versions.yml

```diff
@@ -9,117 +9,200 @@
       ignore_errors: true
       when: "'ubuntu' in ansible_distribution|lower"
 
+    - name: Set Ubuntu version message
+      set_fact:
+        ubuntu_msg: "Ubuntu Version: {{ ubuntu_version.stdout }}"
```

Comment:
A little bit of extra output here (e.g. the tab character) , tidy up only if convenient

```
ok: [doaj-editor-app-1] => {
    "msg": [
        "Ubuntu Version: Description:\tUbuntu 20.04.6 LTS",
```

Reply:
Done